verilog/tools/syntax/verilog_syntax.cc

```diff
@@ -217,8 +269,22 @@ int main(int argc, char** argv) {
       continue;
     }
 
-    int file_status = AnalyzeOneFile(content, filename);
+    Json::Value file_json;
+    int file_status = AnalyzeOneFile(content, filename, file_json);
     exit_status = std::max(exit_status, file_status);
+    if(absl::GetFlag(FLAGS_export_json)) {
+      json[filename] = std::move(file_json);
+    }
   }
+
+  if(absl::GetFlag(FLAGS_export_json)) {
```

Comment:
(Example where `clang-format` will clean up: there is a space missing between `if` and `(`)